demo/runners/support/utils.py

```diff
@@ -115,7 +115,7 @@ def output_reader(handle, callback, *args, **kwargs):
             break
         try:
             run_in_terminal(functools.partial(callback, line, *args))
-        except AssertionError as e:
+        except AssertionError:
```

Comment:
why remove as e?  the error object is useful right, maybe this was some artifact ?

Reply:
The exception is not returning the error. There is just a pass keyword after